jenkins/databricks/params.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
```

Comment:
Make the parameter parsing scripts common for both 'run-build.py' and 'run-test.py'